src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/AdditionalConditionWithConfig.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.AdditionalCondition
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.policy.AdditionalConditions
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.policy.IAdditionalCondition
+
+data class AdditionalConditionWithConfig(
+  val additionalCondition: AdditionalCondition,
+  val config: IAdditionalCondition,
+)
+
+fun mapConditionsToConfig(licenceConditions: List<AdditionalCondition>, policy: AdditionalConditions): List<AdditionalConditionWithConfig> {
```

Comment:
if you make policy a `Set<IAdditionalCondition>`, then you can just pass in `policy.allAdditionalConditions()`